### ISSUE TITLE
Support for tablets

### ISSUE THREAD

**Issue by akkywadhwa** [openMF/mobile-wallet#572]
**Summary:**
Merchants might want to use tablets for shops. Hence, the application should provide support for tablets. 

**Steps to reproduce:**
Install the app in any tablet.

**Expected behavior:**
Adjusted design in tablet

**Observed behavior:**
![Tablet View](https://user-images.githubusercontent.com/666752/69888782-af9f9900-1313-11ea-8974-258bdeef0038.PNG)

**Device and Android version:**
All Tablets

**Screenshots:**
Attached above.

**Comment by jsahil730** [comment]
@akkywadhwa would you like to work on this?

**Comment by akkywadhwa** [comment]
> 
> 
> @akkywadhwa would you like to work on this?

Yes, I have started working on this. Waiting for confirmation from mentor if this is relevant to work on.

**Comment by luckyman20** [comment]
@akkywadhwa I agree to this issue to some extent.

This app's target market is India. Since I live in India and interact with lots of merchant in day-to-day life I find none to be using Tablets (there might be one or two with pre-installed softwares for specific purposes (mostly POS)). I would keep this issue open as of now for community to discuss.

@edcable @naman14 @ankurs287 @shiv07tiwari Thoughts?

**Comment by therajanmaurya** [comment]
Current compose implement support all type of devices
